spec/support/saml_auth_helper.rb

```diff
@@ -417,6 +417,18 @@ def visit_idp_from_ial1_saml_sp(**args)
     saml_authn_request
   end
 
+  def visit_idp_from_saml_sp(saml_overrides: {}, saml_security_overrides: {})
+    settings = saml_settings.dup
+    saml_overrides.each do |setting,value|
+      settings.send("#{setting}=", value)
+    end
+    saml_security_overrides.each do |setting,value|
+      settings.security[setting] = value
```

Comment:
coverage

Reply:
I'll need this moving forward, but for this ticket I'll just remove it.